pages/docs/doc/fipe.json

```
@@ -157,6 +157,65 @@
                     }
                 }
             }
+        },
+        "/fipe/veiculos/v1/{tipoVeiculo}": {
+            "get": {
+                "tags": ["FIPE"],
+                "summary": "Lista os veículos de acordo com a marca e o tipo de veículo",
+                "description": "",
+                "parameters": [
+                    {
+                        "name": "tipoVeiculo",
+                        "description": "Os tipos suportados são `caminhoes`, `carros` e `motos`.\n",
+                        "in": "path",
+                        "required": false,
```

Por ser parte do path me parece ser obrigatório
```suggestion
                        "required": true,
```

---

documentação atualizada.